bigquery_etl/shredder/delete.py

```diff
@@ -261,6 +266,14 @@ def create_job(client):
             ):
                 replace_clause = BUG_1751979_MAIN_V4_REPLACE_CLAUSE
                 temporary_udfs = BUG_1751979_MAIN_V4_UDFS
+            # And corresponding changes for main_summary_v4 which contains values
+            # from one of the affected fields.
+            elif (
+                sql_table_id(target)
+                == "moz-fx-data-shared-prod.telemetry_derived.main_summary_v4"
```

Comment:
Perhaps I should update these conditions to not specify project. I suspect @relud's test may not have picked up these logic changes due to being in a different project.

Reply:
ive been manually changing the project specified here, but maybe i don't have it right. i'll double check what query that job ran.

Reply:
Thank you. I will look more deeply into this.

Reply:
I believe @sean-rose found the root cause; the fix is now merged and @relud kicked off another test.